integration_tests/src/main/python/udf_test.py

```diff
@@ -77,8 +77,8 @@ def pandas_sum(to_process: pd.Series) -> float:
 @allow_non_gpu('AggregateInPandasExec', 'PythonUDF', 'Alias')
 @pytest.mark.parametrize('data_gen', integral_gens, ids=idfn)
 def test_group_aggregate_udf(data_gen):
-    @f.pandas_udf('long')
-    def pandas_sum(to_process: pd.Series) -> int:
+    @f.pandas_udf('double')
```

Comment:
Just FYI:

arrow/spark will convert a long into a double if there are any nulls in a batch, and then cast things back after the processing is done (because of how pandas works).  This caused inconsistencies in a few corner cases (i.e. overflow) for the simple map case when we were not batching things the same on the CPU and GPU.

If these are failing for similar reasons, then it probably has something to do with how we partition the data vs how spark partitions the data, which would then result in different groupings being sent to the GPU, but I am not 100% sure on that.  Just changing it does not solve the underlying difference and we need to understand and document it, because someone may want to process longs or ints and if there are differences we at least need to document them.

Reply:
Thanks Bobby.
These two failures could be probably due to the similar reasons, since the function `max` , which avoids long overflow, works well per the premerge job reports. This changing is just for verification.

As you suggested, i will try to understand where the inconsistencies come from.

Reply:
I am fine with these changes to get us unblocked, because they are testing more then what we were doing right now with a skipped test.  But we still need to understand why we get different results. If we want to do that in a follow on issue I am fine with that, but I want to be sure that we don't lose track of it.